watertap/examples/flowsheets/nf_dspmde/tests/test_nf.py

```diff
@@ -20,13 +20,13 @@
 def test_main():
     m = main()
     test_dict = {
-        "lcow": [m.fs.costing.LCOW, 0.144583],
-        "pressure": [m.fs.NF.pump.outlet.pressure[0] * 1e-5, 5.372779],
-        "area": [m.fs.NF.nfUnit.area, 405.66782957],
+        "lcow": [m.fs.costing.LCOW, 0.141678],
+        "pressure": [m.fs.NF.pump.outlet.pressure[0] * 1e-5, 5.21856],
+        "area": [m.fs.NF.nfUnit.area, 444.423],
         "recovery": [
             m.fs.NF.nfUnit.recovery_vol_phase[0.0, "Liq"] * 100,
-            90,
+            94.64,
```

Comment:
My only concern about letting recovery be "optimized" is that it'll drive up to the upper bound and at least previously, the test solution will deviate by a little every now and then with new changes in future PRs. It may be the case that the latest changes to scaling have remedied the issue completely, but it might also be the case that we'll see future tests fail because the solution converged to another recovery value close to 95% upper bound but outside of the tolerance of the regression test.